pages/ambassadors.tsx

```diff
@@ -0,0 +1,467 @@
+    twitter: "stand_english",
+  },
+  {
+    name: "Wolfmaximus",
+    twitter: "NFTwolfmaximus",
+  },
+].sort((a, b) => a.name.localeCompare(b.name));
+
+const Ambassadors: ThirdwebNextPage = () => {
+  const trackEvent = useTrack();
+
+  return (
+    <DarkMode>
+      <Flex
+        sx={{
+          // overwrite the theme colors because the home page is *always* in "dark mode"
+          "--chakra-colors-heading": "#F2F2F7",
+          "--chakra-colors-paragraph": "#AEAEB2",
+          "--chakra-colors-borderColor": "rgba(255,255,255,0.1)",
+        }}
+        justify="center"
+        flexDir="column"
+        as="main"
+        bg="#000"
+      >
+        <ProductPage
+          seo={{
+            title: "Ambassadors",
+            description:
+              "Join thirdweb's ambassador program to build decentralized technologies, empower developer communities, and build your own brand.",
+            openGraph: {
+              images: [
+                {
+                  url: `${getAbsoluteUrl()}/assets/og-image/sdk.png`,
+                  width: 2334,
+                  height: 1260,
+                  alt: "thirdweb ambassador program",
+                },
+              ],
+            },
+          }}
+        >
+          <HomepageSection pt="100px" bottomPattern>
+            <Flex
+              pt={24}
+              mb={{ base: 24, md: -24 }}
+              flexDir="column"
+              gap={{ base: 6, md: 8 }}
+              align={{ base: "initial", md: "center" }}
+            >
+              <Heading
+                as="h1"
+                fontSize={{ base: "64px", md: "64px" }}
+                fontWeight={700}
+                letterSpacing="-0.04em"
+                mb={4}
+                textAlign="center"
+              >
+                Become a{" "}
+                <Text
+                  fontSize={{ base: "64px", md: "64px" }}
+                  fontWeight={700}
+                  letterSpacing="-0.04em"
+                  as="span"
+                  bgGradient="linear-gradient(243.9deg, #BFA3DA 21.81%, #84309C 48.81%, #C735B0 86.61%);"
+                  bgClip={"text"}
+                >
+                  thirdweb <br /> Ambassador.
+                </Text>
+              </Heading>
+              <Heading
+                as="h3"
+                size="subtitle.md"
+                textAlign="center"
+                maxW="container.sm"
+              >
+                Are you a builder, believer in decentralized technologies, want
+                to empower the developer community, and build your own brand?
+              </Heading>
+              <Flex flexDir="row" gap={3} flexGrow={1} minW={300}>
+                <LinkButton
+                  href="https://docs.google.com/forms/d/e/1FAIpQLSfuhnYtA-CMo5PT1jF4vzzPy0jVS_DNRMSOayV4FNxx7rpKKQ/viewform"
+                  isExternal
+                  onClick={() =>
+                    trackEvent({
+                      category: "cta-button",
```

Comment:
category should be defined at the top with something like `const CATEGORY = "ambassadors_page"` then use throughout the page